core/src/main/java/io/github/mmm/ui/api/widget/input/UiInputContainer.java

```diff
@@ -20,6 +20,18 @@
   /** The {@link io.github.mmm.ui.api.datatype.UiStyles#add(String) style} or element name for an input container. */
   String STYLE = "ui-input";
 
+  /** The style or element name for the suffix container */
+  String STYLE_SUFFIX = "ui-suffix";
+
+  /** The style or element name for the prefix container */
+  String STYLE_PREFIX = "ui-prefix";
+
+  /** The style or element name for the input field attached to a suffix */
+  String STYLE_INPUT_WSUFFIX = "ui-input-field-wSuffix";
+
+  /** The style or element name for the input field attached to a prefix */
+  String STYLE_INPUT_WPREFIX = "ui-input-field-wPrefix";
```

Comment:
Maybe it is even possible to omit these styles (and the code adding and removing them):
With CSS selectors you can do something like
```
.ui-prefix > .ui-input {
  /** this rule applies when an input has a prefix */
}
```
There should also be a possibility for the input followed by a suffix but this is a little more tricky.
Reference documentation:
* https://www.w3schools.com/cssref/css_selectors.asp
* https://developer.mozilla.org/de/docs/Web/CSS/CSS_Selectors